uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeavePeriodEntitlementTest.php

```diff
@@ -65,21 +65,28 @@ public function testOnGetBreakdownDoesNotAcceptPeriodIdWithoutContactId() {
    * @expectedExceptionMessage You must include either the id of a specific entitlement, or both the contact and period id
    */
   public function testGetBreakdownWhenAllParametersArePassed() {
-    civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['entitlement_id'=> 1, 'period_id' => 1, 'contact_id'=>1]);
+    civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['entitlement_id' => 1, 'period_id' => 1, 'contact_id' => 1]);
   }
 
   /**
    * @expectedException CiviCRM_API3_Exception
    * @expectedExceptionMessage You must include either the id of a specific entitlement, or both the contact and period id
    */
   public function testGetBreakdownWhenEntitlentIdAndPeriodIdArePassed() {
-    civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['entitlement_id'=> 1, 'contact_id'=>1]);
+    civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['entitlement_id' => 1, 'contact_id' => 1]);
   }
 
   public function testGetBreakdownWhenContactAndPeriodIdArePassed() {
-    $result = civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['period_id'=> 1, 'contact_id'=>1]);
+    $result = civicrm_api3('LeavePeriodEntitlement', 'getbreakdown', ['period_id' => 1, 'contact_id' => 1]);
     $this->assertEmpty($result['values']);
   }
 
-
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Unable to find a CRM_HRLeaveAndAbsences_BAO_LeavePeriodEntitlement with id 1
+   */
+  public function testGetBreakdownWhenOnlyEntitlementIdIsPassed() {
```

Comment:
This test is not good. The assertEmpty won't even be executed, because the api call will an exception, which will be caught by the @expectedException.

Please use the Fabricator to create a LeavePeriodEntitlement and remove the @expectedException and @expectedExceptionMessage